challenge.html

```diff
@@ -59,8 +69,7 @@ <h2> What we're looking for with this challenge</h2>
                 </br>
                 </br>
                 <h3 class="align-center">Please complete this challenge before: </h3>
-                <h2 class="align-center">September 23 2019</h2>
-
+                <h2 class="align-center">September 30 2020</h2>
```

Comment:
Let's not encourage the software challenge, as we're not recruiting for the aspects that it.

Remove it from the site.